presto-druid/pom.xml

```diff
@@ -16,6 +16,10 @@
     </properties>
 
     <dependencies>
+        <dependency>
```

Comment:
It seems the parent-pom excludes the druid's roaring bitmap which druid requires. The code complained about missing class definition when testing.